examples/nextjs/src/components/my-page.js

```diff
@@ -55,22 +55,27 @@ export function MyPage({ pageAsset, nav }) {
           })
         : DotcmsLayout;
 
+    const pageData = useDotcmsEditorPageAsset(pageAsset);
+    pageAsset = pageData ?? pageAsset;
```

Comment:
Again... we should not do this `useDotcmsEditorPageAsset` give me the right data and I just render.

Reply:
You totally right, fixed!